docs/overview.md

```diff
@@ -0,0 +1,64 @@
+# Interledger Overview
+
+Interledger enables the seamless exchange of value across different payment networks. Traditional payment networks 
+operate independently from each other. Sending value is easy only if the sender and recipient have accounts on the same 
+network, but it can be slow and expensive if they have accounts on different networks.
+
+Interledger is not tied to any one company, blockchain, or currency. You can easily send XRP to someone who wants to 
+receive ETH, or you can send USD through a Paypal account to someone who wants to receive USD in a Venmo account. 
+Interledger makes it easy to transact in whatever currency or payment network you choose.
+
+## What is Interledger?
+Interledger is a network of computers that enables sending value across independent payment networks. Similar to the Internet, 
+Interledger routes packets of value from source to destination. Computers on the Interledger network are 
+called *nodes*. Nodes can take one or more of the following roles: 
+* Sender Node – Initiates a value transfer
+* Router Node – Applies currency exchange and forwards packets of value. This is the intermediary node between 
+the sender and the receiver. 
+* Receiver Node – Receives and completes the value transfer
+
+![ILP-nodes](images/ilp-nodes.png)
+
+## How does Interledger work?
+At the core of Interledger is the [Interledger Protocol (ILPv4)](https://interledger.org/rfcs/0027-interledger-protocol-4/), 
+which is a set of rules that defines how nodes should send value over the Interledger network. ILPv4 is a *request/response* 
+protocol, where each request or response is sent or received in the form of ILP packets. Typically, a single aggregate payment from source to destination... 
+of sending value from source to destination is broken into several ILP packets. Each packet contains transaction 
+information, which is private to the participating nodes. ILPv4 has three packet types -  *Prepare*, *Fulfill*, and *Reject*. 
+
+![ILP-packets](images/ilp-packets.png)
+
+The sender constructs and sends a Prepare packet as a request to the connecting router. The routers forward the packet 
```

Comment:
"routers" or "routing nodes"?

I think the simple definitions are better so I;'d update the list of terms in lines 15 - 17 to just say, sender, receiver and router

Reply:
https://github.com/interledger/interledger.github.io/pull/113/files#diff-f3b56000093113bd3bfb6c9c05e7e945R15